File: FormalBench-data/FilteredData/specifications/seed_726_per_category_10_653ade686f/c/MaxOfTwo.c
#include "java_arrays.h"
#include <stdint.h>

/*@
  assigns \nothing;
  ensures \result == (x > y ? x : y);
*/
int32_t maxOfTwo(int32_t x, int32_t y) {
    return x > y ? x : y;
}

File: FormalBench-data/FilteredData/specifications/seed_726_per_category_10_653ade686f/java/MaxOfTwo.java
import java.io.*;
import java.lang.*;
import java.util.*;
import java.math.*;


class MaxOfTwo {
    
    /*@ public normal_behavior
      @ assignable \nothing;
      @ ensures \result == (x > y ? x : y);
      @*/
    //@ code_java_math
    public static int maxOfTwo(int x, int y) {
        return x > y ? x : y;
    }
}

